Resources/Prototypes/SS220/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -314,3 +314,70 @@
   - type: HeldSpeedModifier
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitCentcom
+
+#ERT Leader Gamma Hardsuit
+- type: entity
+  parent: [ ClothingOuterHardsuitERTLeader ]
+  id: ClothingOuterHardsuitERTLeaderGamma
+  name: Gamma ERT leader's hardsuit
+  components:
+  - type: ExplosionResistance
+    damageCoefficient: 0.3
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.35
+        Slash: 0.35
+        Piercing: 0.25
+        Heat: 0.3
+        Radiation: 0.1  
+        Caustic: 0.4
+        Stamina: 0.1
```

Типа `Stamina` не существует в `DamageSpecifier`
```suggestion
  - type: StaminaResistance
    damageCoefficient: 0.1
```